src/Init/Data/Bool.lean

```diff
@@ -353,16 +353,17 @@ theorem and_or_inj_left_iff :
 /-- convert a `Bool` to a `Nat`, `false -> 0`, `true -> 1` -/
 def toNat (b : Bool) : Nat := cond b 1 0
 
-@[simp] theorem toNat_false : false.toNat = 0 := rfl
+@[simp, bv_toNat] theorem toNat_false : false.toNat = 0 := rfl
 
-@[simp] theorem toNat_true : true.toNat = 1 := rfl
+@[simp, bv_toNat] theorem toNat_true : true.toNat = 1 := rfl
```

Comment:
Why would a theorem in Bool be part of `bv_toNat`? I would drop this here.

Reply:
@tobiasgrosser because it allows `bv_omega` to decide things that are to do with bitvectors. Concretely, due to the addition of theorems like `concatBit` which take a `(b: Bool)` as argument, it's super duper useful to have this be part of `bv_omega`. Morally, a boolean is a width 1 bitvector, so I'm comfortable saying that this falls in its purview.

Reply:
But does `bv_omega` not run the std simp-set as well as part of its loop?

Reply:
Nope!